lib/legacy/legacy-element-mixin.js

```diff
@@ -47,7 +53,8 @@ export const LegacyElementMixin = dedupingMixin((base) => {
    * @extends {HTMLElement}
    * @private
    */
-  const legacyElementBase = DirMixin(GestureEventListeners(ElementMixin(base)));
+  const legacyElementBase = builtCSS ? GesturesElement :
```

Comment:
The unconditional `@implements {Polymer_DirMixin}` annotation _should_ be fine here?